Python/jit.c

```diff
@@ -278,6 +278,7 @@ patch(unsigned char *base, const Stencil *stencil, uintptr_t patches[])
             case HoleKind_ARM64_RELOC_GOT_LOAD_PAGE21:
             case HoleKind_IMAGE_REL_ARM64_PAGEBASE_REL21:
             case HoleKind_R_AARCH64_ADR_GOT_PAGE:
+            case HoleKind_ARM64_RELOC_BRANCH26:
```

Comment:
This is where I hit a wall - basically I hit `./jit_stencils.h:249:12: error: use of undeclared identifier 'HoleKind_ARM64_RELOC_BRANCH26'` after touching up the build files as it looks like there's a case not handled.

I'm not confident that this is where it belongs. This segfaults right now and I am unfamiliar with C debugging.

Reply:
Close! This is the MachO version of ELF's `R_AARCH64_JUMP26`/`R_AARCH64_CALL26` relocation (basically a 28-bit relative branch). So it can go up with those instead.

And, to predict the future, I'm guessing you might get assertion errors on 244 and 245 after you do that. That's because some of the jumps we're making are more than 28 signed bits away, so we need to extend them with a "trampoline". That's already set up for ARM Linux, thankfully, so it *should* fix the issue if you just add `"ARM64_RELOC_BRANCH26"` to the set near the top of `StencilGroup.process_relocations`.

Reply:
If you want we can also go over C debugging sometime too.